projects/addon-commerce/components/thumbnail-card/thumbnail-card.style.less

```diff
@@ -122,35 +123,24 @@
     transform-origin: right;
 }
 
+.t-content {
+    position: absolute;
+    bottom: 0;
+}
+
 .t-icon {
     inline-size: 1rem;
     block-size: 1rem;
 }
 
-.t-front {
-    .fullsize();
-
-    display: flex;
-    align-items: flex-end;
-    backface-visibility: hidden;
-    box-shadow: inset 0 0 0 1px var(--tui-background-neutral-1);
-    border-radius: inherit;
-}
-
-.t-back {
+// card backface
+:host::after {
```

Comment:
Let's move it under `:host`

Reply:
https://github.com/taiga-family/taiga-ui/pull/9612/commits/0259cb128af54bc8421d196028603c2ef07fc60c